recipe/bld.bat

```diff
@@ -3,8 +3,7 @@ REM Install headers with projectConfig.cmake files with cmake
 mkdir build
 cd build
 cmake -G "NMake Makefiles" ^
-      -D CMAKE_INSTALL_PREFIX=%PREFIX% ^
-      -D USE_PYTHON_INCLUDE_DIR=ON ^
```

Comment:
I just digged a bit deeper to check what this is doing in conjunction with the python (helper) package below.

If we remove this line,
```python
import pybind11

pybind11.get_include()
```
will not return the right path anymore.

We might want to implement (patch) an external path hint for the include in `setup.py` to avoid breaking downstream packages that query this.

https://github.com/pybind/python_example/issues/8